src/components/NavBar/navBar.module.css

```diff
@@ -17,19 +17,10 @@
 
 .dropDefault {
   color: #FF007f;
-  text-align: center;
   text-decoration: none;
   display: inline-block;
 }
 
-a.dropdown-toggle {
-  color: #ffffff;
-}
-
-a.dropdown-toggle:hover{
-  color: black;
-}
-
```

Comment:
This has moved to `src/components/NavBar/navBar-static.css`. It didn't work previously because CSS Modules generates different class names to avoid conflicts. Here, we WANT to override the class name originally provided by Bootstrap.